mtp_api/apps/performance/migrations/0004_retire_performance_platform.py

```diff
@@ -0,0 +1,33 @@
+from django.db import migrations
+
+
+def remove_performance_platform_update(apps, schema_editor):
+    cls = apps.get_model('core', 'ScheduledCommand')
+    cls.objects.filter(name='update_performance_platform').delete()
+
+
+def restore_performance_platform_update(apps, schema_editor):
+    cls = apps.get_model('core', 'ScheduledCommand')
+    cls.objects.create(
```

Comment:
these 2 are here because i wanted there to be a record _somewhere_ of the current scheduled tasks